src/AutoBuild.jl

```diff
@@ -1301,6 +1347,14 @@ function build_jll_package(src_name::String,
         if lazy_artifacts
             println(io, "using LazyArtifacts")
         end
+
+        if !isempty(augment_platform_block)
+            print(io, """
+            include(path) = Base.include(@__MODULE__, path)
+            include(joinpath("..", ".pkg", "platform_augmentation.jl"))
```

Comment:
Just to understand, this is needed because in a bare module `include` is not defined?  A comment about this may help future readers :slightly_smiling_face: 

Do we need to define this function (is `include` used elsewhere?) or can just do
```suggestion
            Base.include(@__MODULE__, joinpath("..", ".pkg", "platform_augmentation.jl"))
```
?

Reply:
Yeah!